src/Microsoft.ComponentDetection.Orchestrator/Microsoft.ComponentDetection.Orchestrator.csproj

```diff
@@ -8,9 +8,6 @@
         <PackageReference Include="Newtonsoft.Json" />
         <PackageReference Include="Polly" />
         <PackageReference Include="Serilog" />
```

Comment:
Required?

Reply:
Yes, since we use it to dynamically update the minimum logging level from `Orchestrator.cs`